### PR TITLE
added bananacheck command

### PR DESCRIPTION
**Pull request by corey-b**
**`!bananacheck @user`** (should) let you check how many bananas another user has.

I don't know how to test the bot though.

### CODE REVIEW

File: bananacheck.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+const Discord = require("discord.js");
+
+module.exports.run = async (bot, message, args, db, guild) => {
+  console.log(`Running command ${this.cmd.name}`);
+  const config = require("../config.json");
+  let whom = args[1]
```
Comment (astrajs) [inline bananacheck.js:6]: args[1] is going to be the second parameter. in this case, you'd want args[0].

args[1] is:
!bananacheck [0] [1]

**Review by astrajs — CHANGES_REQUESTED** [review]
line 6
let whom = args[0]

**Comment by astrajs** [comment]
i had to re-read the code a couple of times to actually understand what you're trying to do 😆 

the code wouldn't function as intended, because there's no feature written to check other's banana values. even the embed description is copy pasted straight from the !banana command.

`.setDescription(`Banana collected! Come back in ${timeLeftString} for more.`)`

**Comment by Miodec** [comment]
Im not sure about this one, could lead to people getting pinged a lot

**Pull request closed without merging**